wit/types.wit

```diff
@@ -244,7 +244,23 @@ interface types {
 
     /// Gives the `incoming-body` associated with this request. Will only
     /// return success at most once, and subsequent calls will return error.
+    ///
+    /// Deprecated in favor of `into-parts`.
```

Comment:
I'm not sure we need to mark `consume` as *deprecated* in 0.2.1  -- if all you want is to do is read the body stream, it's a reasonable method to call (and perhaps a bit more efficient; not forcing you to copy strings you don't want).  Note that 0.3 is free and expected to make big breaking changes, and if you check out #101, you can see a sketch of how it might look.

Reply:
Removed these notes in 55a4754